helm/prometheus-rules/templates/alerting-rules/cilium.rules.yml

```diff
@@ -17,10 +17,10 @@ spec:
       expr: avg(cilium_bpf_map_pressure) by (cluster_id, map_name) * 100 > 80
       for: 15m
       labels:
-        area: kaas
+        area: managedservices
```

Comment:
Is that not platform? I'm confused

Reply:
i based this on other alerts and there were `managedservices` used

Reply:
I do not think you're wrong, we need to sort those labels :)